setup.sh

```diff
@@ -52,3 +52,9 @@ sudo rm /var/swap
 # speed up boot
 sudo apt purge exim4-* nfs-common triggerhappy
 sudo apt --purge autoremove
+
+
+# ensure we don't override kernel option for 'ondemand' frequency
+# governor
+sudo systemctl disable loadcpufreq.service
+sudo systemctl disable raspi-config.service
```

Comment:
Good one! This does enable ondemand (super logical name for such a service :man_facepalming: ).

Can you use `mask` instead of `disable` to ensure that it really doesn't get started?

Reply:
yup!